### PR TITLE
feat added alert component

### PR DESCRIPTION
**Pull request by Himanshu-Dhawale**
Screenshot 2022-02-11 235215

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]

File: docs/container.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/* container */
+
+.container {
+    position: fixed;
+    top: 4.5rem;
+    left: 16.94rem;
+    width: calc(100vw - 16.94rem);
+    height: calc(100vh - 5rem);
+    padding: 0 1.5rem;
+    z-index: 0;
```
Comment (Maianki) [inline docs/container.css]: By default, z-index is zero you can probably remove this.

File: docs/container.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    margin-bottom: 1rem;
+    text-align: center;
+    font-size: 1rem;
+  }
+  .container_components {
+    padding: 1rem;
+    margin-bottom: 1rem;
+    background: var(--clr-white);
+    height: auto;
+    /* max-width: 60rem; */
```
Comment (Maianki) [inline docs/container.css]: remove commented code.

File: docs/sidebar.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    margin: 1rem 1rem 1rem 0;
+    /* border-bottom: 1px solid #c1c7d0; */
+    font-size: 1.25rem;
+  }
+  
+  .side_name {
+    font-weight: 900;
+    font-size: 1.10rem;
+  }
+
```
Comment (Maianki) [inline docs/sidebar.css:57]: remove commented code.

**Review by Maianki — APPROVED** [review]
suggested a few changes, everything else looks good.

**Pull request merged**